openapi/specs/v1.yaml

```diff
@@ -3258,6 +3258,66 @@ paths:
               schema:
                 $ref: '#/components/schemas/LearningResourceSearchResponse'
           description: ''
+  /api/v1/learning_resources_user_subscription/:
+    get:
+      operationId: learning_resources_user_subscription_list
+      description: View for listing percolate query subscriptions for a user
+      tags:
+      - learning_resources_user_subscription
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/UserPercolateQueryRequest'
+          description: ''
+  /api/v1/learning_resources_user_subscription/{id}/unsubscribe/:
+    post:
+      operationId: learning_resources_user_subscription_unsubscribe_create_2
+      description: View for listing percolate query subscriptions for a user
+      parameters:
+      - in: path
+        name: id
+        schema:
+          type: string
```

Comment:
Should this be `int` ?

Reply:
this shouldnt be there at all at. just pushed a fix that includes proper schema